x/slashing/keeper/signing_info.go

```diff
@@ -72,6 +72,62 @@ func (k Keeper) SetValidatorMissedBlocks(ctx sdk.Context, address sdk.ConsAddres
 	store.Set(types.ValidatorMissedBlockBitArrayKey(address), bz)
 }
 
+// Get a boolean representing whether a validator missed a block with a specific index offset
+func (k Keeper) GetValidatorMissedBlockBitFromArray(bitGroupArray []uint64, index int64) bool {
```

Comment:
This could even be a simple util/helper function.

GetBoolean (bitArray, index)
SetBoolean(bitArray, index)